src/components/AlbumInfo/Buttons.jsx

```diff
@@ -0,0 +1,37 @@
+
+
+export const Buttons = () => {
```

Comment:
Really good idea to put all the code for the buttons in a separate component for a cleaner look across the board! The only thing I could think of as a tip for next time is that you could just add the svgs in an other folder (public) and then link to them, so that you wouldn't have the need to put the svg-code here 🥰